### DISCUSSION TITLE
Does this library support attachments?

### DISCUSSION THREAD

**Discussion by Dr-Harris** [dolanmiu/docx#2390] [Q&A]
![微信截图_20231023101338](https://github.com/dolanmiu/docx/assets/22830171/0a5cc78b-3d1d-4e00-99cf-31b0e2719359)

**Comment by dolanmiu** [comment]
Does not yet nope, sorry

**Reply by Dr-Harris** [reply]
Is there a support plan for the future?

**Reply by dolanmiu** [reply]
Probably not. AFAIK attachments is a Microsoft Word specific feature?

Do other word processors support this? i can't find any info
